card/mod/02_basic_types/set/all/rss.rb

```diff
@@ -6,11 +6,10 @@
 
   def initialize card, args
     super
-    @xml = @parent.xml if @parent
+    @xml = @parent ? @parent.xml : ::Builder::XmlMarkup.new
```

Comment:
this seems like a clear improvement.  RssFormat objects always have @xml objects.